Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help clean clean-test clean-pyc clean-build help lint test playwright-shiny playwright-examples playwright-deploys install-trcli install-playwright
+.PHONY: help clean% check% format% help lint test playwright-shiny playwright-examples playwright-deploys install-trcli install-playwright
```

Add `pyright` and clean up `playwright` and `install`
```suggestion
.PHONY: help clean% check% format% help lint test pyright playwright% install%
```